marketplacetests/firefox_accounts/app.py

```diff
@@ -18,15 +21,19 @@ class FirefoxAccounts(Base):
     _password_input_locator = (By.ID, 'password')
     _sign_in_button_locator = (By.ID, 'submit-btn')
     _firefox_logo_locator = (By.ID, 'fox-logo')
+    _not_you_logout_link_locator = (By.CSS_SELECTOR, '.logout')
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
+        self.change_to_correct_login_frame()
+
+    def change_to_correct_login_frame(self):
         self.marionette.switch_to_frame()
-        self.wait_for_element_present(*self._ff_accounts_frame_locator)
-        self.frame = self.marionette.find_element(*self._ff_accounts_frame_locator)
-        self.marionette.switch_to_frame(self.frame)
-        self.wait_for_element_not_present(*self._body_loading_locator)
-        self.wait_for_element_displayed(*self._firefox_logo_locator)
+        time.sleep(1)
```

Comment:
We really try to avoid using `sleep`. Can you define a wait that will work instead, e.g., wait for either frame to be present?